props/saber_fett263_buttons.h

```diff
@@ -5792,24 +5799,33 @@ SaberFett263Buttons() : PropBase() {}
 #endif
           return true;
         }
-        if (menu_) MenuDial(-1);
-        return true;
+        if (menu_) {
+          MenuDial(-1);
+          return true;
+        }
+        return false;
 
       case EVENTID(BUTTON_NONE, EVENT_TWIST_RIGHT, MODE_OFF):
         if (wav_player && wav_player->isPlaying()) {
           current_menu_angle_ = fusor.angle2();
           return true;
         }
-        if (menu_) MenuDial(1);
-        return true;
+        if (menu_) {
+          MenuDial(1);
+          return true;
+        }
+        return false;
 
       case EVENTID(BUTTON_NONE, EVENT_TWIST_LEFT, MODE_OFF):
         if (wav_player && wav_player->isPlaying()) {
           current_menu_angle_ = fusor.angle2();
           return true;
         }
-        if (menu_) MenuDial(-1);
-        return true;
+        if (menu_) {
+          MenuDial(-1);
+          return true;
+        }
+        return false;
```

Comment:
I guess this eats modifier buttons if you return true?

Reply:
It was in some of the Serial Monitor outputs users shared, if you were moving the hilt in a way that it detected the turn it would get in the way of other events like clashes, etc.